Kernel/Devices/HID/PS2KeyboardDevice.cpp

```diff
@@ -74,6 +74,7 @@ void PS2KeyboardDevice::irq_handle_byte_read(u8 byte)
                 g_io_work->queue([this, ch]() {
                     ConsoleManagement::the().switch_to(ch - 0x02);
                 });
+                key_state_changed(ch, pressed);
```

Comment:
all cases of this switch and the surrounding if call key_state_changed in the end, so this could probably be written more neatly as just:
```c++
if ((m_modifiers & Mod_Alt) != 0 && ch >= 2 && ch <= ConsoleManagement::s_max_virtual_consoles + 1) {
    g_io_work->queue([this, ch]() {
        ConsoleManagement::the().switch_to(ch - 0x02);
    });
}
key_state_changed(ch, pressed);
```

Reply:
Done in next revision.